examples/src/main/java/nl/jcore/java8demo/streaming/StreamingExecutionPlanExample.java

```diff
@@ -9,7 +9,7 @@ public static void main(String[] args) {
         final List<Integer> numbers = Arrays.asList(1, 2, 3, 4, 5, 6, 7, 8);
         final List<String> output = numbers.stream()
                 .filter(i -> i % 2 == 0)
-                .sorted((i1, i2) -> i2.compareTo(i1))
+                .sorted((i1, i2) -> i2.compareTo(i1)) // Alternative: .sorted(Comparator.reverseOrder())
```

Comment:
Ah, okay. "_reverseOrder_" sounds like it will reverse the current order. But I guess it actually applies "_reversedNaturalOrder_".